### PR TITLE
feature/change smtp consumer

### PR DESCRIPTION
**Pull request by borovHokage**

### CODE REVIEW

File: src/EmailService.Broker/Helpers/Email/EmailResender.cs [hunk 1]
```diff
@@ -38,7 +36,7 @@ public async Task StartResend(int intervalInMinutes, int maxResendingCount)
     public EmailResender(
       IUnsentEmailRepository unsentEmailRepository,
       ILogger<EmailResender> logger,
-      IRequestClient<IGetSmtpCredentialsRequest> rcGetSmtpCredentials)
+      ISmtpSettingsRepository rcGetSmtpCredentials)
```
Comment (MariyaSin) [inline src/EmailService.Broker/Helpers/Email/EmailResender.cs]: rename the variable

File: src/EmailService.Broker/Helpers/Email/EmailSender.cs [hunk 2]
```diff
@@ -15,7 +13,7 @@ public class EmailSender : BaseEmailSender
 
     public EmailSender(
       ILogger<EmailSender> logger,
-      IRequestClient<IGetSmtpCredentialsRequest> rcGetSmtpCredentials,
+      ISmtpSettingsRepository rcGetSmtpCredentials,
```
Comment (MariyaSin) [inline src/EmailService.Broker/Helpers/Email/EmailSender.cs]: rename the variable

File: src/EmailService.Broker/Helpers/Email/EmailResender.cs (hunk 1 above)
Comment (borovHokage) [reply src/EmailService.Broker/Helpers/Email/EmailResender.cs]: fixed

File: src/EmailService.Broker/Helpers/Email/EmailSender.cs (hunk 2 above)
Comment (borovHokage) [reply src/EmailService.Broker/Helpers/Email/EmailSender.cs]: fixed

**Review by MariyaSin — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Review by ekaterinaryzhenkova — APPROVED** [review]

**Review by asyagraf — APPROVED** [review]

**Pull request merged**